hack/push-and-tag.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
My mind broke on this script. I thought this would process `$OUTPUT_TASK_BUNDLE_LIST` where image references in the form of `quay.io/konflux-ci/task-xyz:0.1-task-hash@sha256:hash`, remove the `-task-hash` and push, i.e. release the task. I must be missing something.